### PR TITLE
Fix issues with JWT, icons and subscriptions 

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
This PR fixes a lot of issue with the test app:
- 41799e5: Since https://github.com/mattermost/mattermost-plugin-apps/pull/268 apps need to request the use of JWT. The app paniced as it was expecting the JWT header. It now properly requests JWT.
- 092d18a: Unifies the app style with the examples in the apps plugin
- 4967780: Fix a pointer issue to make bindings show up again
- ec5babe: The `/test subscribe` command now works again. Support for all subscriptions was added. Some don't work (yet) as there is no `ActingUserID`.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-39645
https://mattermost.atlassian.net/browse/MM-39692

### CODE REVIEW

File: routers/mattermost/bindings/channelheader/invalid.go [hunk 1]
```diff
@@ -2,59 +2,52 @@ package channelheader
...
 
-func getWithNoLabel(siteURL, appID string) *apps.Binding {
-	icon := utils.GetIconURL(siteURL, "icon.png", appID)
-
-	return &apps.Binding{
+func getWithNoLabel() apps.Binding {
+	return apps.Binding{
 		Location: "ERROR_with_no_label",
-		Icon:     icon,
+		Icon:     "icon.png",
```
[the remark this answers is not in the document]
Comment (hanzei) [reply routers/mattermost/bindings/channelheader/invalid.go:32]: Yes, the Apps Framework takes care of that.

[the remark this answers is not in the document]
Comment (mickmister) [reply routers/mattermost/bindings/channelheader/invalid.go:32]: @DHaussermann Can we please verify this through manual (or e2e) testing? We would just need to run this app with a server configured with a subpath, and verify that the icons are displayed properly.

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- No Panic observed when installing with JWT
- Subscription test commands are working
- No issues found 

LGTM! 
Please merge.

**Comment by DHaussermann** [comment]
@hanzei note that I don't have access to change labels. Reviews are complete.

**Pull request merged**